### PR TITLE
Feat/featured section

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Modified the homepage banner to use it to showcase featured content.

## Type of change

- Enhancement (a non-breaking change that adds functionality)

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

## To do 

* validate slide props
* test across devices

### CODE REVIEW

File: src/common/components/FeaturedSlider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+    return (
+        <Flickity
+            options={flickityOptions} // takes flickity options {}
+        >
+            <RenderSlides/>
+        </Flickity>
+    )
+}
```
Comment (Dr-Electron) [inline src/common/components/FeaturedSlider/index.tsx:68]:
```suggestion
}

```

**Review by Dr-Electron — COMMENTED** [review]
LGTM. But why are we linking to just Shimmer tutorials from the IOTA landing page? 😅 

We should then at least switch to the shimmer wiki to make it clear by the color scheme that the user just switched environment. But even then...

**Comment by lucas-tortora** [comment]
> LGTM. But why are we linking to just Shimmer tutorials from the IOTA landing page? sweat_smile
> 
> We should then at least switch to the shimmer wiki to make it clear by the color scheme that the user just switched environment. But even then...

When I started this feature a couple of weeks ago, the wikis were still one. That being said, we'll probably have more featured content in shimmer than in IOTA in the upcoming months.  For the time being, I've updated the PR to showcase IOTA tutorials only.

I think it'd be a good idea to limit this PRs scope to adding the featured slider component, and I can create subsequent PR to add a Shimmer featured section.  WDYT?

File: src/common/components/FeaturedSlider/slides.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  {
+    "title": "Send IOTA Tokens with Javascript",
+    "description": "Learn how to create your seed, addres, send IOTA Tokens and retrieve your balance in the Devnet.",
+    "imagePath": "/img/homepage/resources_bot.svg",
+    "solidButton": {
+      "buttonText": "Start tutorial",
+      "url": "/tutorials/send-iota-tokens-with-javascript"
+    }
+  }
+]
```
Comment (Dr-Electron) [inline src/common/components/FeaturedSlider/slides.json:24]:
```suggestion
]

```

**Review by Dr-Electron — APPROVED** [review]

**Review by jlvandenhout — APPROVED** [review]

**Pull request merged**